OrbitGl/App.cpp

```diff
@@ -239,6 +242,27 @@ void OrbitApp::PostInit() {
 
   string_manager_ = std::make_shared<StringManager>();
   GCurrentTimeGraph->SetStringManager(string_manager_);
+
+  if (absl::GetFlag(FLAGS_enable_tracepoint_feature)) {
```

Comment:
reverse the if

if (!absl::GetFlag(...)) {
  return;
}

...

it makes code more readable: https://engdoc.corp.google.com/eng/doc/tott/episodes/455.md?cl=head
(please let me know if you do not have access to it)

Reply:
I do not have access to it, but I will reverse the if.